docs/apworld_dev_faq.md

```diff
@@ -43,3 +43,26 @@ A faster alternative to the `for` loop would be to use a [list comprehension](ht
 ```py
 item_pool += [self.create_filler() for _ in range(total_locations - len(item_pool))]
 ```
+
+---
+
+### I learned about indirect conditions in the world API document, but I want to know more. What are they and why are they necessary?
+
+The world API document mentions indirect conditions and **when** you should use them, but not *how* they work and *why* they are necessary. This is because the explanation is quite complicated.
+
+Region sweep (the algorithm that determines which regions are reachable) is a Breadth First Search of the region graph from the Menu region, checking entrances one by one and adding newly reached nodes (regions) and their entrances to the queue until there is nothing more to check.
+
+However, if entrance access conditions depend on regions, then it is possible for this to happen:
+1. An entrance that depends on a region is checked and determined to be untraversable because the region hasn't been reached yet during the graph search.
```

Comment:
```suggestion
1. An entrance that depends on a region is checked and determined to be nontraversable because the region hasn't been reached yet during the graph search.
```
English sucks lol
Non- means something along the lines of negating, null, etc. Nontraversable means it cannot be traversed.
Un- means basically the opposite of it. Untraversable means it cannot be traversed, but with a more permanent feel? Like, it's the opposite of traversable -- it will make you go the other way (in a vibes sense).
look it's dumb but I think the change makes sense and it makes me happy